SQL/0000-00-00-schema.sql

```diff
@@ -359,6 +359,7 @@ CREATE TABLE `instrument_subtests` (
   `Subtest_name` varchar(255) NOT NULL default '',
   `Description` varchar(255) NOT NULL default '',
   `Order_number` int(11) NOT NULL default '0',
+  UNIQUE KEY `Subtest_name` (`Subtest_name`),
```

Comment:
I don't think this constraint makes sense. Subtest_name is the name of the page, but different instruments might have pages with the same name. (ie. different instruments might both have different pages both named "page2")

The constraint should be on the combination of `(Test_name, Subtest_name)`